cmake/modules/RootConfiguration.cmake

```diff
@@ -676,7 +676,7 @@ get_filename_component(ROOT_BINDIR \"\${_thisdir}/${ROOT_CMAKE_TO_BIN_DIR}\" ABS
 # Deprecated value ROOT_BINARY_DIR
 set(ROOT_BINARY_DIR_SETUP "
 # Deprecated value, please don't use it and use ROOT_BINDIR instead.
-get_filename_component(ROOT_BINARY_DIR \"\${_thisdir}/${ROOT_CMAKE_TO_BIN_DIR}\" ABSOLUTE)
+get_filename_component(ROOT_BINARY_DIR \"\${ROOT_BINDIR}\" ABSOLUTE)
```

Comment:
Isn't this going to be the directory one level up from what you want?

Reply:
Could you just use `message(STATUS )` to print both variables after calling find_package(ROOT) and paste the result here, please?

Reply:
message(STATUS ${ROOT_BINARY_DIR})
message(STATUS ${ROOT_BINDIR})
=================================
-- /home/oksana/CERN_sources/root-cxxmodules/builds
-- /home/oksana/CERN_sources/root-cxxmodules/builds/bin

Reply:
Shouldn't they both be the same?

Reply:
I suppose that was from *within a ROOT build*, and not as-if-external-project?

Reply:
@amadio @Axel-Naumann yes it was from within ROOT, from outside (standalone roottest) it is:

 oksana@oksana-dellxps  ~/CERN_sources/roottest/builds   fix_root_binary_dir  cmake ..
...
-- /home/oksana/CERN_sources/root-cxxmodules/builds/bin
-- /home/oksana/CERN_sources/root-cxxmodules/builds/bin